pkg/controller/common/keystore/resources_test.go

```diff
@@ -101,6 +101,13 @@ func TestResources(t *testing.T) {
 
 set -eux
 
+keystore_initialized_flag=/bar/data/elastic-internal-init-keystore.ok
```

Comment:
This isn't necessarily your change but it seems odd we're not reusing the `script` constant here